doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_memory.rst

```diff
@@ -42,19 +42,20 @@ Local RAM is present in each of local domains
 Application core RAM
 --------------------
 
-.. image:: images/nrf54h20_memory_map_app.svg
-   :width: 300 px
+.. table:: Application core RAM layout in the nRF54H20
+
+   +-------------+-----------+---------+----------+-----------------------------+
+   |Start Address|End Address|Size (KB)|Total (KB)|Content                      |
+   +=============+===========+=========+==========+=============================+
+   |config       |0x22008000 |config   |32        |Application's Non-Secure data|
```

Comment:
I think we should define in this document what "config" means. (it means configurable by an application developer to meet the application needs).